stories/card/card.stories.js

```diff
@@ -574,10 +561,11 @@ export const tableCard = () => `
 </div>
 `;
 
+tableCard.storyName = 'Table card';
 tableCard.parameters = {
     docs: {
         iframeHeight: 400,
-        storyDescription: 'The Header and the Content sections follow the specification described in the Card Anatomy. The Table component belongs to the Content area. The <code>.fd-card--table</code> modifier class needs to be applied with <code>.fd-card</code>.'
+        storyDescription: 'Cards can display tables within the content area. To add a table, add the <code>fd-card--table</code> modifier class to the main element.'
```

Comment:
"To add a table, add the <code>fd-card--table</code> modifier class to the main element." 

The way it's written I'm afraid that the user might think that this modifier class will add a table to the card. This modifier class is needed to turn a standard card to a Table card and apply some additional styling. Something like:
"To render a Table card, add the <code>fd-card--table</code> modifier class to the main element." I think will make more sense (feel free to modify it, it's just to explain what I mean :) )

Reply:
Oops, you're totally right about this one!